backend/flow_api/handler.go

```diff
@@ -81,6 +84,36 @@ func (h *FlowPilotHandler) validateSession(c echo.Context) error {
 				continue
 			}
 
+			if h.Cfg.Session.ServerSide.Enabled {
+				// check that the session id is stored in the database
+				sessionId, ok := token.Get("session_id")
+				if !ok {
+					lastTokenErr = errors.New("no session id found in token")
+					continue
+				}
+				sessionID, err := uuid.FromString(sessionId.(string))
+				if err != nil {
+					lastTokenErr = errors.New("session id has wrong format")
+					continue
+				}
+
+				sessionModel, err := h.Persister.GetSessionPersister(nil).Get(sessionID)
+				if err != nil {
+					return fmt.Errorf("failed to get session from database: %w", err)
+				}
+				if sessionModel == nil {
+					lastTokenErr = fmt.Errorf("session id not found in database")
+					continue
+				}
+
+				// Update lastUsed field
+				sessionModel.LastUsed = time.Now().UTC()
```

Comment:
`updatedAt` should be updated automatically by pop. I added the later one, because IMO the two have semantically different meanings. One is updated when the session was used and one when the model was updated, this does not need to be the same.

Reply:
I think pop uses the timezone which is used on the machine. So when its set to UTC it is using UTC. But when that is the case, we must set the `UpdatedAt` everywhere, which we did not do, I think.